scripts/start-docker.sh

```diff
@@ -11,7 +11,18 @@ uid=$(ls -nd . | awk '{ print $3 }')
 
 # Create an `olympia` user with that ID, and the current directory
 # as its home directory.
-useradd -Md $(pwd) -u $uid olympia
+if [[ $uid -eq 0 ]]; then
+    # Don't try and create a user with uid 0 since it will fail.
+    # Works around issue with docker for mac running containers
+    # as root and not the user. Instead we just create the olympia
+    # user since files will transparently be still owned by the host
```

Comment:
"files will transparently be still owned" is awkwardly phrased, I think "will still be owned" would do the trick?